tests/functional/products/vpn/test_landing.py

```diff
@@ -10,26 +10,28 @@
 @pytest.mark.nondestructive
 def test_vpn_available_in_country(base_url, selenium):
     page = VPNLandingPage(selenium, base_url, params='?geo=us').open()
-    assert page.is_try_vpn_hero_button_displayed
+    assert page.is_try_vpn_hero_fixed_price_button_displayed
```

Comment:
Naming convention: Should this be `get` instead of `try`?

Reply:
Good suggestion!